### ISSUE TITLE
Блокирование изменения настроек панели

### ISSUE THREAD

**Issue by mikhailnov** [mikhailnov/gpo-analytics#5]
У нас в https://github.com/mikhailnov/gpo-analytics/blob/main/KDE/Первый_запуск.md описан механизм создания конфига `plasma-org.kde.plasma.desktop-appletsrc`. Администраторы часто спрашивают, как запретить пользователю менять, например, панель задач. По моему опыту, пользователи действительно часто ее ломают, а потом сами не могу починить.

Нам надо двигаться в сторону приложения того, что описываем, к реальным потребностям.

Можно ли сделать такой конфиг, чтобы пользователь не мог менять панель? Может быть, заблокировать виджет без возможности разблокировать?

**Comment by VictorR2007** [comment]
Как я понял, ты не про панель задач, а про панель.
Панель задач. это часть панели, не котором располагаются ярлыки работающих приложений.

**Comment by VictorR2007** [comment]
Нужно поискать, но я никогда не слышал и сам не сталкивался с возможностью заблокировать изменение панели.

**Comment by mikhailnov** [comment]
Да, про всю панель. Может, можно с помощью знаков $ в конфигах как-то. Еще бы изменение обоев рабочего стола блокировать, например. Или, может, как-то сразу вообще все настройки можно заблокировать?

**Comment by VictorR2007** [comment]
Посмотри рабочий файл. Он постоянно меняется, и там конкретные пути.
Этот файл даже не описывается в политиках, так как там его невозможно использовать.
[plasma-org.kde.plasma.desktop-appletsrc.tar.gz](https://github.com/mikhailnov/gpo-analytics/files/11961682/plasma-org.kde.plasma.desktop-appletsrc.tar.gz)

**Comment by mikhailnov** [comment]
Я в курсе, в файле Первый_запуск об этом и написал. Но ты где-то приводил пример блокировки всей секции настроек, а не конкретного параметра. Может, как-то так можно что-то заблокировать.

**Comment by mikhailnov** [comment]
Можно ли взять существующий конфиг ~/.config/plasma-org.kde.plasma.desktop-appletsrc, в него добавить знаки доллара и положить в /etc/xdg, тем самым запретив пользователю его менять? Можешь проверить и задокументировать?

**Comment by VictorR2007** [comment]
> Можно ли взять существующий конфиг ~/.config/plasma-org.kde.plasma.desktop-appletsrc, в него добавить знаки доллара и положить в /etc/xdg, тем самым запретив пользователю его менять? Можешь проверить и задокументировать?

Нельзя. Я уже писал об этом.
Этот файл предназначен только для того пользователя, для которого он был создан.
Это не универсальный файл с настройками, а динамический, только для текущего пользователя.
Специально выше приложил файл, где видно, что всё идёт от пользователя victor.

Может знаешь способ полной блокировки панели для XFCE или может Gnome?
Чтобы подсмотреть, как там это реализовано.

**Comment by mikhailnov** [comment]
Предположим, что на компе только один пользователь или что настройки панели у всех пользователей одинаковые. UUID тоже можно сделать одинаковыми, думаю, если очень надо.
